src/templates/filter-bar.jsx

```diff
@@ -61,4 +63,8 @@ export const FilterBar = React.createClass({
     },
 });
 
-export default FilterBar;
+const mapStateToProps = state => ({
+    filterName: getFilterName(state),
+});
+
+export default connect(mapStateToProps, null)(FilterBar);
```

Comment:
since `mapDispatchToProps` is optional, we don't need to pass `null` in, we can make it a single-arg function call

```js
connect( mapStateToProps )( FilterBar );
```